screens/PersonaSetup.tsx

```
@@ -0,0 +1,152 @@
+
+            <TouchableOpacity 
+                style={requiredFieldsComplete ? [styles.button, styles.buttonEnabled] : [styles.button, styles.buttonDisabled]}
+                onPress={handlePersonaSetup}
+                disabled={!requiredFieldsComplete}
+                accessibilityRole="button"
+                accessibilityLabel="Continue to profile setup"
+                accessibilityHint="Continues to profile setup once terms and privacy policy are accepted"
+                accessibilityState={{ disabled: !requiredFieldsComplete }}
+            >
+                <Text style={requiredFieldsComplete ? styles.buttonText : [styles.buttonText, styles.buttonDisabledText]}>
+                    I'm ready to be Vested
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)